conftest.py

```diff
@@ -361,7 +361,7 @@ def imported_vm(host, vm_ref):
     else:
         vm_orig = host.import_vm(vm_ref, host.main_sr_uuid(), use_cache=CACHE_IMPORTED_VM)
 
-    if CACHE_IMPORTED_VM:
+    if CACHE_IMPORTED_VM or is_uuid(vm_ref):
```

Comment:
In what is it linked to SB tests?
I'm not sure this is a behaviour we want.  It will likely break users' expectations, and such changes should come with a rationale of why we'd want to do that.
Couldn't this usecase be fulfilled by the "import_vm: add clone:// and clone+start:// URIs" commit in the `install` branch?

Reply:
It's mostly for convenience. If the `install` branch already has this feature, I will remove it.